File: Seb4Vision.CSportView.Data.Model/DataTransferObject/PlayerDTO.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Seb4Vision.CSportView.Data.Model.DataTransferObject
{
    public class PlayerDTO
    {
        public long PlayerID { get; set; }
        public long TeamID { get; set; }
        public string Name { get; set; }
        public string Surname { get; set; }
        public string PreferredName { get; set; }
        public string JerseyNumber { get; set; }
        public string PhotoPath { get; set; }
        public float Weight { get; set; }
        public float Height { get; set; }
        public string Position { get; set; }
        public string PositionShortDesc { get; set; }
        public string PersonType { get; set; }
        public string PersonTypeShort { get; set; }
        public MatchEventDTO PlayerEvents { get; set; }

        public PlayerDTO()
        {
            PlayerEvents = new MatchEventDTO();
        }
    }
}

File: Seb4Vision.CSportView.Web/ClientApp/app/models/playerDTO.ts
import { MatchEventDTO } from "./matchEventDTO"; 
import { SportVuPlayerStatDTO } from "./sportvuplayerstatsDTO";

export interface playerDTO {
    playerID: number;
    teamID: number;
    name: string;
    surname: string;
    teamfullname: string;
    preferredName: string;
    jerseyNumber: string;
    photoPath: string;
    weight: number;
    height: number;
    position: string;
    positionShortDesc: string;
    personType: string;
    personTypeShort: string;
    playerEvents: MatchEventDTO;
    sportVuPlayerStats: SportVuPlayerStatDTO;
}